ndonnx/additional/__init__.py

```diff
@@ -1,12 +1,13 @@
 # Copyright (c) QuantCo 2023-2024
 # SPDX-License-Identifier: BSD-3-Clause
 
-from ._additional import fill_null, make_nullable, isin, shape, static_map
+from ._additional import fill_null, make_nullable, isin, shape, static_map, static_shape
```

Comment:
Do we need to export `static_shape` publicly? When would one use it rather than `Array.shape`?

Reply:
I considered leaving it private but I think the convention of having all overridable functionality being standalone public functions is valuable when contributing custom dtypes. On second thought we can always make it public later so I'll make it private for this release and then revisit this later.